GeneralsMD/Code/GameEngine/Source/GameNetwork/NetPacket.cpp

```diff
@@ -32,6 +32,15 @@
 #include "GameNetwork/networkutil.h"
 #include "GameNetwork/GameMessageParser.h"
 
+// TheSuperHackers @refactor BobTista 10/06/2025 Extract magic character literals into named constants for improved readability
+namespace NetPacketFieldTypes {
+	const char COMMAND_TYPE = 'T';		// NetCommandType field
+	const char RELAY = 'R';				// Relay field
+	const char PLAYER_ID = 'P';			// Player ID field
+	const char COMMAND_ID = 'C';		// Command ID field
+	const char FRAME = 'F';				// Frame field
+	const char DATA = 'D';				// Data payload field
```

Comment:
I suggest use PascalCase because MACRO_CASE is reserved for preprocessor macros and always has the potential for causing name conflicts.